audio/format.c

```diff
@@ -79,32 +73,37 @@ static const int planar_formats[][2] = {
     {AF_FORMAT_DOUBLEP, AF_FORMAT_DOUBLE},
 };
 
+bool af_fmt_is_planar(int format)
+{
+    for (int n = 0; n < MP_ARRAY_SIZE(planar_formats); n++) {
+        if (planar_formats[n][0] == format)
+            return true;
+    }
+    return false;
+}
+
 // Return the planar format corresponding to the given format.
-// If the format is already planar, return it.
-// Return 0 if there's no equivalent.
+// If the format is already planar or if there's no equivalent,
+// return it.
 int af_fmt_to_planar(int format)
```

Comment:
have you checked whether this behaviour change breaks another assumption done by callers?

Reply:
> af_fmt_to_planar is not used anywhere now. Let me know if I should just ditch it.